web/client/src/workers/sqlglot/worker.ts

```diff
@@ -1,16 +1,17 @@
-import 'https://cdn.jsdelivr.net/pyodide/v0.22.1/full/pyodide.js'
-
 export {}
```

Comment:
I think we don't need ti anymore , if we not using `type: 'module`

Reply:
Good point. I'll remove it.